### ISSUE TITLE
[Feature request] Mouse aim outside of item use and bindable mouse buttons + Free mouse camera

### ISSUE THREAD

**Issue by AQDuck97** [Mr-Wiseguy/Zelda64Recomp#183]
I'm not talking about free camera here, just when you C-up (or look through the telescope) it would be nice if this also worked with mouse aim. I don't know if looking is different codewise to aiming, but that seems highly unlikely.

And bindable mouse buttons would be a nice finishing touch.

**Comment by SvenBudak** [comment]
That's exactly what I thought when I walked past the shooting range for the first time... I'll never be able to do that with the keyboard :D Mouse support would be really great for aiming with the bow/slingshot.

**Comment by AQDuck97** [comment]
> That's exactly what I thought when I walked past the shooting range for the first time... I'll never be able to do that with the keyboard :D Mouse support would be really great for aiming with the bow/slingshot.

Shooting range supports it, it's just when you look without equipment.
Aced both ranges thanks to it

**Comment by JamesK48** [comment]
Agreed, aiming with the bow using WASD is awful and mouse aiming and attacking with left mouse button would be a great addition too!

**Comment by AQDuck97** [comment]
Actually now that v1.1 added free camera, I'm adding that to the wishlist

**Comment by Reonu** [comment]
This, along with mouse free camera support, is a planned feature

**Comment by baaannnaaannn** [comment]
maybe this project will help https://github.com/Graslu/1964GEPD

**Comment by Sol1vaN** [comment]
still this is not available right?
i've tested the last build and this have a slide control for mouse sens, but the mouse don't respond yet
also i can't bind mouse buttons

**Comment by kevinwilhoit** [comment]
Mouse buttons (left click, right click) need to be available for binding, full stop. Issue for many with physical limitations
